### DISCUSSION TITLE
Welcome to lnd Discussions!

### DISCUSSION THREAD

**Discussion by Roasbeef** [lightningnetwork/lnd#5572] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stillwoods** [comment]
Roasbeef,

I was just about to upgrade my Raspberry Pi lightning node.  I downloaded lnd-linux-armv7-v0.13.1-beta.tar.gz and verified signatures and sha256 of the binary files lncli and lnd as I have done in the past and they all checked out.  But when I  ls -al  lnd and lncli (just before copying the binaries into the appropriate folder), I noticed the date of these binary files are:  Dec. 31, 2019 which is prior to the dates of the version 0.12.0 binaries I already had installed.   What's up with this?   I did not install because of this puzzling earlier dated binary files though all the other verifications checked out.   Can you confirm that these binaries are okay and that that date is perhaps an oversight or is something else going on?

**Reply by guggero** [reply]
The date is set to a specific date to make the builds reproducible. Since we want both the binary _and the packaged `.tar.gz`_ to be reproducible we need to manually set the date to a static value. Otherwise everyone building the packages will get a different hash for the `.tar.gz`.

**Reply by stillwoods** [reply]
Isn't it a little confusing to manually set the date of those binaries to a date prior to the date existing on the previous version binaries?  Could that date manually be set to something after the date of the previous binary versions?  I understand not now for the existing v0.13.1, but I mean for future version releases.

**Reply by guggero** [reply]
Yeah, I guess we could take the date of the last commit or something like that.
